keyboards/1upkeyboards/super16v2/v1_1/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

Comment:
Doing this at the keyboard level is wrong — it would break configurator builds (currently the configurator still does not support encoder maps).  The `ENCODER_MAP_ENABLE` option can be used only at the keymap level.

Because of that, it's not really possible to add a separate keyboard revision just for VIA encode map changes — the update needs to be done in place, but with changing the USB PID at the same time, so that the updated firmware version can use a different VIA JSON.